sdk/digitaltwins/digital-twins-core/review/digital-twins-core.api.md

```diff
@@ -99,14 +99,17 @@ export class DigitalTwinsClient {
     listModels(dependeciesFor?: string[], includeModelDefinition?: boolean, resultsPerPage?: number, options?: OperationOptions & PageSettings): PagedAsyncIterableIterator<DigitalTwinsModelData, DigitalTwinModelsListResponse>;
     listRelationships(digitalTwinId: string, options?: OperationOptions & PageSettings): PagedAsyncIterableIterator<any, DigitalTwinsListRelationshipsResponse>;
     publishComponentTelemetry(digitalTwinId: string, componentName: string, payload: string, messageId: string, options?: OperationOptions): Promise<RestResponse>;
-    publishTelemetry(digitalTwinId: string, payload: any, messageId: string, options?: OperationOptions): Promise<RestResponse>;
+    publishTelemetry(digitalTwinId: string, payload: any, // eslint-disable-line @typescript-eslint/explicit-module-boundary-types -- changing the type any would be a breaking change
```

Comment:
Oh wow! Did not realize that the api extractor would pull the comment out...

Just for kicks, can you try using the `eslint-disable-next-line` instead of `eslint-disable-line`, pull the comment to a line above the line causing the linter error, build the package and see if api extractor still wants to pull this comment?

Reply:
Yeah, I'll give it a shot

Reply:
Same problem occurs.

Reply:
Not really! The changes to the api view file are gone, so this is good!

Reply:
The pipeline failed so I regenerated the files again